content/millennium/r4/scheduling/appointment.md

```diff
@@ -231,6 +231,34 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 <%= disclaimer %>
 
+### Example - Update reasonCode
+
+#### Request
+
+    PATCH https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Appointment/4627745
+
+#### Body
+
+<%= json(:r4_appointment_reasoncode_patch) %>
+
+#### Response
+
+<%= headers status: 200 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Tue, 26 Mar 2019 15:42:29 GMT
+Etag: W/"1"
+Last-Modified: Tue, 26 Mar 2019 15:42:27 GMT
+Vary: Origin
+X-Request-Id: 47306a14c8a2c3afd4ab85aa9594101d
```

Comment:
Need to capture a sandbox 200 response and update X-Request-Id and Last-Modified.

Reply:
conveniently we put a note on here indicating that examples are non-normative, so we can get away with making up pretend x-request-id and last-modified values :)

Reply:
Are you implying that we don't need to update this information, then?

Reply:
correct, we do not need to update those two fields